lib/Optimizer/CodeGen/VerifyQIRProfile.cpp

```diff
@@ -9,7 +9,7 @@
 #include "PassDetails.h"
 #include "cudaq/Optimizer/Builder/Intrinsics.h"
 #include "cudaq/Optimizer/CodeGen/Passes.h"
-#include "cudaq/Optimizer/CodeGen/Peephole.h"
+#include "cudaq/Optimizer/CodeGen/QIRFunctionNames.h"
```

Comment:
Clarifying question - why this change?

Reply:
The verifying doesn't need the peephole patterns (it doesn't rewrite the code), but it was including the peephole header to get some function names. It should've included the names directly.